File: frontend/src/app/dto/company/company.update.res.dto.ts
import { CompanyUpdateDataResDto } from "./company.update.data.res.dto";

export class CompanyUpdateResDto{
    private _data! : CompanyUpdateDataResDto
	private _message! : string

    public get data(): CompanyUpdateDataResDto {
        return this._data;
    }

    public set data(data: CompanyUpdateDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }
    
}

File: backend/src/main/java/com/lawencon/ticketms/dto/company/CompanyUpdateResDto.java
package com.lawencon.ticketms.dto.company;

public class CompanyUpdateResDto {
	private CompanyUpdateDataResDto data;
	private String message;
	public CompanyUpdateDataResDto getData() {
		return data;
	}
	public void setData(CompanyUpdateDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
